packages/design-system/src/components/MultiSelectField/style.scss

```diff
@@ -6,6 +6,7 @@ $base-class: 'multiselect';
   position: relative;
   width: inherit;
   color: $font-primary-color;
+  font-size: 14px;
```

Comment:
is it 14px in ds? please consult with @jo

Reply:
yup, it's on all designs

Reply:
ok, I talk with @Jolama, should be 15 😢

Reply:
done